### PR TITLE
implemented style selector

### PR DESCRIPTION
**Pull request by nannobug**

### CODE REVIEW

File: src/overview/Overview.jsx [hunk 1]
```diff
@@ -5,35 +5,45 @@ import AddToCart from './AddToCart.jsx';
...
+      this.setState({ selectedStyle: this.props.styles.results[0] });
+    }
+  }
+
+  handleSelect(style) {
+    this.setState({ selectedStyle: style });
+  }
+
+  render () {
+    console.log(this.state.selectedStyle);
```
Comment (AndrewGunnCho) [inline src/overview/Overview.jsx]: Let's comment this out or remove it to keep the code clean~

File: src/overview/OverviewInformation.jsx [hunk 2]
```diff
@@ -1,15 +1,41 @@
...
+    this.state = {};
+  }
+
+  render () {
+    var product = this.props.product;
+    var style = this.props.selectedStyle;
+
+    let price;
+    if (style.sale_price !== null) {
+      console.log(`Sales price is ${style.sale_price}`);
```
Comment (AndrewGunnCho) [inline src/overview/OverviewInformation.jsx]: Let's comment this out or remove it to keep the code clean~

**Comment by nannobug** [comment]
removed console.log;
add more styling

**Review by AndrewGunnCho — APPROVED** [review]
LGTM!

**Pull request merged**